packages/visx-geo/src/projections/Projection.tsx

```diff
@@ -149,7 +152,11 @@ export default function Projection<Datum extends GeoPermissibleObjects>({
 
   const path = geoPath().projection(currProjection);
 
-  if (pointRadius) path.pointRadius(pointRadius);
+  // @ts-expect-error Pending `@types/d3-geo` to add digits method (introduced in 3.1.0)
+  // @see https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/67363
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-call
+  if (digits !== undefined) path.digits(digits);
+  if (pointRadius !== undefined) path.pointRadius(pointRadius);
```

Comment:
Side suggestion / question: https://github.com/airbnb/visx/pull/213/files#r1388679342

Reply:
I think this change is fine (even though `0` shouldn't be a valid input)